src/components/NotificationError.tsx

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import classNames from 'classnames';
+import React, { useEffect, useState } from 'react';
+import { TodoError } from '../types/TodoError';
+import { wait } from '../utils/fetchClient';
+
+type Props = {
+  errorMessage: TodoError,
+  clearError: () => void,
+};
+
+export const NotificationError: React.FC<Props> = ({
+  errorMessage,
+  clearError,
+}) => {
+  const [isHidden, setIsHidden] = useState(true);
+
+  const handleDeleteClick = () => {
+    setIsHidden(true);
+    wait(1000).then(() => clearError());
+  };
+
+  useEffect(() => {
+    setIsHidden(false);
+    wait(3000)
+      .then(() => setIsHidden(true))
+      .then(() => wait(1000).then(() => clearError()));
```

Comment:
I don't understand a lot of ```wait```ings. Is it really necessary?

Reply:
It's not a lot) The first one is needed to automatically hide the notification box after 3 seconds and the second one is needed to unset the message since the notification box fades away smoothly and we don't want the message to disappear instantly.